rust/azure_iot_operations_protocol/src/rpc_command/executor.rs

```diff
@@ -1433,7 +1610,18 @@ async fn handle_ack(
                 Ok(ack_ct) => {
                     match ack_ct.await {
                         Ok(()) => log::info!("[pkid: {pkid}] Acknowledged"),
-                        Err(e) => log::error!("[pkid: {pkid}] Ack error: {e}"),
+                        Err(e) => {
+                            match e {
+                                azure_iot_operations_mqtt::error::CompletionError::Detatched => {
```

Comment:
is this a typo in the underlying error or just here? If underlying, can you fix that in this PR if it's small?

Reply:
This is directly from the mqtt client so the fix would have to be there